### ISSUE TITLE
Full Auto and Assassin Dice

### ISSUE THREAD

**Issue by TheOrgg** [buttonmen-dev/buttonmen#2493]
These two have similar first parts of their ability-- double the value on a die that's attacking, unbounded by maximum die size.  Since the 'overshot' value isn't permanent, that shouldn't require a visual representation of the effect happening.

The difference is what happens afterwards.  Full Auto you lose a turn.  Assassin, the points scored from the captured die are halved.

Full Auto can be used in Power or Skill attacks.  So I'm guessing "Full Auto Power Attack" and "Full Auto Skill Attack" would be options for the attacks if a Full Auto Die was selected.

Assassin Dice have a special 'Assassin Attack'

Technically the Assassin die cuts the value in half of the other die, but I couldn't find an example when just simply doubling the value wasn't the same end result since the halved value is rounded up.

**Comment by AdmiralJota** [comment]
Capturing Value dice? There are probably other skill combinations where it might be relevant too.

**Comment by jl8e** [comment]
Armor dice are another interaction where doubling vs halving could matter. (Possibly, depending on whether the interaction is interpreted as modifying the armored value or the base value. (Base value would be my off-the-cuff interpretation. ))

Fire dice are another where it matters. If you have 9, and want a 20, doubling means you need two points of fire, but halving means only one.

**Comment by TheOrgg** [comment]
Value dice-- Halfing it would cause the Assassin Attack vs a value die to be worse.  That is a different outcome, fair enough.  Without finding a method of contacting Mr. Shelley, we'd have to argue about intent vs rule.

Can Fire Dice affect non-Power/Skill attacks?  I don't think so.  It'd only be applicable to Full Auto, which optionally doubles the value when used in a Power or Skill attack.

An Assassin attack, the one that originally cut the attacked die's value in half, is an independent attack like a Speed attack, thus Fire Dice couldn't be used.  

Here's the rules for Assassin Dice verbatim from the set's origin, spelling errors left intact:  "When Assassin Dice are used in an attack they cut the size of the number showing on the opponents [sic] die in half (rounding fractions up). The captured die is now only worth half its value in points (rounding up) when scoring."

That's pretty messy, and if taken literally it means it'd convert the die it attacked into a Value Die as well (but I'm pretty sure that wasn't the intent).  My rephrasing tried to capture the intent, a separate attack to use the ability and full usage of Power and Skill attacks.  Only being able to Assassin Attack with it would make it kinda suck, as it loses a lot of points (or if going by value, a majority of points) each use.

Let us make a chart for Armor dice...
(charts not make-able here... lemme wiki page it)  I didn't even need to finish the chart.  Halving it against armor dice (assuming the base value on the die is used to calculate the half, as I feel is reasonable) makes the Assassin Attack weaker.  
https://buttonmen.fandom.com/wiki/Scratchzilla_figures_and_arguing

Another argument for changing it to doubling is that it is easier to do the math in your head, but that might just be me.  Doing the math with the armor dice in my head was actually slightly confusing.

**Comment by TheOrgg** [comment]
I did think of a question with Full Auto dice-- if you use two of them in a skill attack against Billy Shakespeare (or your Full Auto d20s rolled a 3 and a 2 and you skill attack a ten...), do you lose two turns or one?

Gut feeling says one.  Former M:TG Judge in me says two.

**Comment by jl8e** [comment]
As written,  there's no such thing as an assassin attack, and it does kick in all the time., which makes skill attacks weird

**Comment by jl8e** [comment]
It's not great, but I think it's clear enough to derive coherent meaning. It's far from certain that that's the desinger''s intent, but that's a different problem. It's nothing like as bad as, say, Evil or Teleport.

My interpretation: when one or more assassin dice are used to attack, calculate the attack as if the target is showing half the value showing on the die. Dice captured by assassin dice have their value halved. This applies to all attack types, since it doesn't specify just power, or just skill, or power and  skill. It's also not optional, since there's nothing to say it is.

The big question unanswered is how it applies to multi-target attacks (speed, berserk,  rush), and I think halving everything is the only consistent answer.

This does make using assassin dice a huge headache for the player, but that's another problem.

**Comment by TheOrgg** [comment]
Yeah, Evil and Teleport were a mess.  I think I figured them out.  Teleport could be fun...  or run into the Winslow problem.  Evil is pretty evil.  Gaming Guardians was obviously not tested, and I'm fine with that being the last Die Skill added to the site.  A few of the buttons could be implemented somewhat painlessly, though.  In custom buttons, some of the GGard's skills would be interesting.

You prefer a literal interpretation, I favor a more player-centric adaptation with the basic end result the same (and it happens to clean up the messy what-if's since it only happens when the attack is used).  Would you also prefer the attacked die to score equal to half the value on the die, too, as the rules are written?

Hell, what I'd really prefer is getting into contact with Mr. Shelley who designed the set.  I've got a feeler out via the person who might be the artist for the set, so I might be able to contact him via that-- he can clarify what the correct rule is.

**Comment by jl8e** [comment]
Mostly, I disagree with the claim that the end result is the same. The ability described is always in effect, and, because of the scoring, that matters.

Also, your reading of the scoring part of the power incorrectly conflates 'value shown on the die' with 'value of the die in points'. It's clear, especially since the rest of the power description uses 'number' when referring to the value shown in the die.

**Comment by dwvanstone** [comment]
(Context for this thread: Assassin dice are in Order of the Dolls. (See #1530). )

I like:

> My interpretation: when one or more assassin dice are used to attack, calculate the attack as if the target is showing half the value showing on the die. Dice captured by assassin dice have their value halved. This applies to all attack types, since it doesn't specify just power, or just skill, or power and skill. It's also not optional, since there's nothing to say it is.

I'd add a few "(rounded up)"s in there. Is there a source that says that Assassin dice can't do a power attack or skill attack and only have an Assassin attack? It seems from the source text that whenever they are used in any kind of attack their special skill is used.

If we don't get a response from Mr. Shelley I think we can do whatever we want with respect to "half the value" or "half the size" when scoring. 


I'd be more interested in the Full Auto skill as suggested above since MegaTokyo has artwork and descriptions. 

> I did think of a question with Full Auto dice-- if you use two of them in a skill attack against Billy Shakespeare (or your Full Auto d20s rolled a 3 and a 2 and you skill attack a ten...), do you lose two turns or one?

I'd think you lose two turns since you have to "reload" for each time you double a die.

**Comment by TheOrgg** [comment]
I actually heard back from Cinco Barnes, and he's forwarding Mr. Shelley my information.  I'm hoping to hear from him.

If I do, this is the one big question I'd like to ask.  I'd like to have them on the side of more playable than as a weakness on a button.

And I'm trying to reach out to the person who did the MegaTokyo set, but I need someone who plays League of Legends to contact the user via that platform.

**Comment by TheOrgg** [comment]
I finally was able to talk to Reah about his intent for the Assassin die.  He thinks doubling works well for it.  Also, they can't use Power or Skill attacks as a normal die, they must be used as an Assassin die.

https://buttonmen.fandom.com/wiki/Order_of_Dolls

**Comment by TheOrgg** [comment]
Played with my local friend today and we encountered a question about Assassin Dice.  As we've been playing with the skill, whenever it attacks the value doubles (any kind of attack) and the amount of points the dice captured are worth get halved.  

Well, one of the dice captured was a Poison d12.  Does that make it worth negative three points, or would it make it worth negative twelve points?  I've sent this to Rhea to see what he says, but it's an interesting situation for sure.


We also had Socrates versus Fluttershy who had her Rage die used to capture her Rebound die and ended up rebounding her own Rage die.  It was kinda funny and neat.
